sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -581,4 +569,19 @@ public CryptographyClientBuilder disableChallengeResourceVerification() {
 
         return this;
     }
+
+    /**
+     * Disables the ability to perform cryptographic operations locally, performing all cryptographic operations on the
+     * service side instead.
+     *
+     * <p>This method will have no effect if used in conjunction with the
```

Comment:
We should throw an exception when we see such conflicting build configuration. Silently ignoring will result in customers assuming that local cryptography is disabled when it's actually not.